src/Payum/Stripe/StripeCheckoutGatewayFactory.php

```diff
@@ -16,7 +16,7 @@ class StripeCheckoutGatewayFactory extends GatewayFactory
      */
     protected function populateConfig(ArrayObject $config)
     {
-        if (!class_exists('Stripe')) {
```

Comment:
replace "stripe/stripe-php":"2.0 - 3.6" with "stripe/stripe-php:~2.0|~3.0" so a user can copy and past it to composer console  command.